### DISCUSSION TITLE
drop() got multiple values for argument 'axis'

### DISCUSSION THREAD

**Discussion by kzaho** [ourownstory/neural_prophet#1220] [General]
F-n check_dataframe at (https://github.com/ourownstory/neural_prophet/blob/main/neuralprophet/df_utils.py) fails.
Is there an extra star symbol, which should be deleted?
```
regressors_to_remove = list(set(regressors_to_remove))
        checked_df = checked_df.drop(*regressors_to_remove, axis=1)
```

**Comment by LeonieFreisinger** [comment]
@kzaho The regressors you want to use need to be added with `.add_lagged_regressor()`. After instantiating the model, you can not drop regressors anymore. In that case, you need to re-instantiate your model.

**Reply by judussoari** [reply]
Solved in PR #1272
